compiler/damlc/tests/src/unstable-types.sh

```diff
@@ -32,7 +32,6 @@ for LF_VERSION in $PKG_DB/*; do
         stdlib=$LF_VERSION/daml-stdlib-*.dalf
         prim=$LF_VERSION/daml-prim.dalf
         $DIFF -b -u <(get_serializable_types $stdlib) <(cat <<EOF
-"DA.Upgrade:MetaEquiv"
```

Comment:
This is a typeclass with only a single method which was therefore serializable but is now translated to Unit.